amazon/instancemetadata/fake/build.gradle.kts

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    testImplementation(project(path = ":http4k-connect-amazon-s3"))
```

Comment:
S3 included in test artifacts to show the ec2 credentials provider authorizing requests for another module